test/e2e/serve.json

```diff
@@ -1,4 +1,15 @@
 {
   "public": "build",
+  "headers": [
+    {
+      "source": "**",
+      "headers": [
+        {
+          "key": "content-security-policy",
+          "value": "default-src none"
+        }
+      ]
+    }
+  ],
```

Comment:
@cherniavskii By any chance do you know how to make the CSP header work? I want to add it to e2e but I can't seem to make it fail :|

I've benchmarked the cost of the eval check: if there is a blocking CSP, it's consistently 1ms on each call. I don't like adding the cost, if we can make the e2e work.

Reply:
Nvm, I'll add a meta tag.